{"uuid":"201be164-1b94-489b-8bec-158730dc9805","historyId":"cf5027aec97a6630091c88ddff3e4bcb","fullName":"ge.tbc.testautomation.tests.SelenideTests2.validateOrderMechanics","labels":[{"name":"package","value":"ge.tbc.testautomation.tests.SelenideTests2"},{"name":"testClass","value":"ge.tbc.testautomation.tests.SelenideTests2"},{"name":"testMethod","value":"validateOrderMechanics"},{"name":"parentSuite","value":"TestSuite"},{"name":"suite","value":"Selenide Tests"},{"name":"subSuite","value":"ge.tbc.testautomation.tests.SelenideTests2"},{"name":"host","value":"tekla"},{"name":"thread","value":"11988@tekla.TestNG-tests-1(20)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Validate order mechanics for pricing page"},{"name":"feature","value":"Books Functionality"},{"name":"epic","value":"UI Validation"},{"name":"severity","value":"critical"}],"links":[],"name":"validateOrderMechanics","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Total Price mismatch! expected [1699.0] but found [8070.25]","trace":"java.lang.AssertionError: Total Price mismatch! expected [1699.0] but found [8070.25]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:969)\r\n\tat org.testng.Assert.assertEquals(Assert.java:640)\r\n\tat ge.tbc.testautomation.steps.PricingSteps.validateTotalPrice(PricingSteps.java:242)\r\n\tat ge.tbc.testautomation.tests.SelenideTests2.validateOrderMechanics(SelenideTests2.java:72)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\r\n\tat org.testng.TestRunner.run(TestRunner.java:587)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:28)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:842)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Screenshot on Failure","source":"e11b397b-fcf6-4b88-b4bc-bd4d6096b1cb-attachment","type":"image/png"}],"parameters":[],"start":1734602429776,"stop":1734602438365}